### ISSUE TITLE
Analytics tab minor updates

### ISSUE THREAD

**Issue by sglangevin** [medic/medic-webapp#900]
I'd like to make a few changes on the Analytics tab to make things easier for the user to see. Here's a mockup for discussion: https://beta.moqups.com/joshnesbit/OYsa1SrpdY/view

**Comment by abbyad** [comment]
@sglangevin do you have a final set of changes? Can you post them here when you do and I'll remove the `Needs Design Work` label so we can start the dev.

**Comment by sglangevin** [comment]
The mockup I posted earlier is final - I created it with Amanda and @diannakane added some comments - we talked through it together. It's ready to go!

**Comment by abbyad** [comment]
Ok, great. @garethbowen if you run into any design questions let us know.

**Comment by garethbowen** [comment]
The "visits completed so far" widget has been removed - is this intentional?

**Comment by garethbowen** [comment]
Unfortunately we can't make the graphs any wider and still have it work on small devices.

**Comment by garethbowen** [comment]
With the "weeks pregnant" field I've left it as is. We can't change the title depending on the the configuration because the configuration may allow for LMP and non-LMP forms to be submitted. I couldn't think of a title that worked for both. Currently there's a "~" indicator for non-LMP submissions in an attempt to indicate the number is approximate. Suggestions welcome!

**Comment by garethbowen** [comment]
With the "CHW" field you wanted to link to their interactions. We can easily link to Reports submitted by this CHW, or Messages submitted by this CHW, or we could link to the CHW in the Contacts tab. Would any of these do?

Anything more than that would require some design work as it's a new screen so probably outside the scope of this issue.

**Comment by sglangevin** [comment]
Submitted a few issues related to data displaying in Analytics - #949 and #950 

As to your last question, @garethbowen I am fine with the CHW field linking to reports submitted by that CHW. Just a search in the reports tab for all reports they have submitted across all patients.

The visits completed so far widget was removed intentionally. And I'm not sure how to acceptance test the "~" indicator because the one person that I have registered that popped up into the upcoming appointments widget doesn't have any weeks pregnant listed.
